test/backtrace.jl

```diff
@@ -183,6 +183,7 @@ let bt, found = false
 end
 
 # issue 28618
+Base.CoreLogging.with_logger(Base.CoreLogging.NullLogger()) do
```

Comment:
Is this meant to suppress the output of `@info` below? If so, why not using `@test_logs` instead which actually checks the log is working as expected (although this one in particular isn't incredibly interesting)?

Reply:
```suggestion
@test_logs begin
```